### DISCUSSION TITLE
What's the purpose of our project?

### DISCUSSION THREAD

**Discussion by ktmitton** [ktmitton/Mittons.Fixtures#96] [Ideas]
* Why would people want to use this?
* What makes this different from docker compose? Why is it better (or worse)?
* Can this be used for documentation?

**Comment by Rauell** [comment]
I can immediately think of a couple ways this differs from docker-compose:
- Using C#, we have the ability to have more fine-tuned controls over startup processes.
- The fixtures have the ability to declaratively state all dependencies, both those that are managed by the fixtures and those that are unmanaged (such as proxies). 
- The fixtures are not limited to a specific implementation, such as docker. These dependencies can be a mixture of different sources.
